doc/source/datainputs.rst

```diff
@@ -16,28 +16,128 @@ To simplify the examples below, we will assume in all cases that the user wishes
     import freud
     rdf = freud.density.RDF(bins=50, r_max=5)
 
+Native Integrations
+===================
+
+The **freud** library offers interoperability with several popular tools for particle simulations, analysis, and visualization.
+Below is a list of file formats and tools that are directly supported as "system-like" objects (see :class:`freud.locality.NeighborQuery.from_system`).
+Such "system-like" objects are data containers that store information about a periodic box and particle positions.
+Other attributes, such as particle orientations, are not included automatically in the system representation and must be loaded as separate NumPy arrays.
 
 GSD Trajectories
-================
+----------------
 
-Using the GSD Python API, GSD files can be very easily integrated with **freud** as shown in :ref:`gettingstarted`. This format is natively supported by `HOOMD-blue <https://hoomd-blue.readthedocs.io/>`_ and is recommended for use with freud.
+Using the GSD Python API, GSD files can be easily integrated with **freud** as shown in :ref:`gettingstarted`.
+This format is natively supported by `HOOMD-blue <https://hoomd-blue.readthedocs.io/>`__ and is recommended for use with freud because it also stores other quantities, such as orientations, that are required by some analysis methods in **freud**.
```

Comment:
I think we should remove the recommendation, it gets the sequence wrong. If people are doing simulations that require orientation information, they will use simulation engines and data formats that support that information, and freud can process it. However, freud also works for other data, and one of our major selling points is being agnostic to the data source.